src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,26 @@
+.stars {
+  padding: 0;
+  display: flex;
+  list-style: none;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin: 2px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:first-child {
+    margin-left: 0;
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
+
+  &__star:nth-child(-n + 4) {
```

Comment:
Consider using css from the previous task to keep modifiers working

Reply:
Not fixed